public/dist/js/manageForum.js

```diff
@@ -1,4 +1,4 @@
-$('.deleteAccount').click(function () {
+$('.deleteForum').click(function () {
```

Comment:
do you mean delete thread?

Reply:
Actually, it is the event listener for both deleting threads and comments at the same time. I could not come up with a better name but it is indeed confusing. I'll rename it to `deleteForumItem` such that it does not suggest that it deletes the entire forum.